moveit_plugins/moveit_ros_control_interface/CMakeLists.txt

```diff
@@ -76,10 +76,11 @@ EXPORT export_${PROJECT_NAME}
 ARCHIVE DESTINATION lib
 LIBRARY DESTINATION lib
 RUNTIME DESTINATION bin
+INCLUDES DESTINATION include
```

Comment:
Do we actually need this .?

Reply:
I wasn't able to get the expected result without that, but I haven't dug into why exactly that is yet.

Reply:
I checked this again, and it looks like it's actually the other way around. The `install` function needs to have the `INCLUDES DESTINATION include` line for the headers to be found by other packages.

Reply:
afaik, this doesn't really install the headers but it only makes them available for the target. This should not be necessary, if the headers are actually installed in the right path, adding this is line is still correct.